rfc/001-composable-pipelines/README.md

```diff
@@ -97,12 +97,6 @@ store<Arrow, Void>
 
 This results in a `<Void, Void>` pipeline that does not "leak".
 
-Assuming that there won't be other types at the logical level besides `Void` and
-`Arrow`, we can simplify the typing discussion to a binary model: either data
-flows (= `Arrow`) or not (= `Void`). This would be desirable as it reduces
-complexity and obviates the need of thinking about typing at the logical level.
-The first implementation will reveal whether this is feasible.
-
```

Comment:
The `Matcher` type is mentioned in this document, so we have more than 2 logical types

Reply:
Duly noted.